test/resources/HistoricSample_AuditEvents.json

```diff
@@ -272,6 +272,6 @@
             "type": "automaton.contentSentToCustomer",
             "engagementID": "187286680131967188"
         },
-        "generatedAt": "2021-03-02T13:23:44Z"
+        "generatedAt": "2021-03-02T13:23:44.868Z"
     }
 ]
```

Comment:
Should we add new line at the end of the file?

Reply:
done